vscode-extension/src/extension.ts

```diff
@@ -221,6 +197,49 @@ class ErrorCentralPanel {
     });
   }
 
+  private _handleBlob(data: any, filePath: string) {
+    // to do make these unique across multiple sessions
```

Comment:
Standard practise is to have "to do" always written `TODO`, as most IDE's will recognize this and highlight, plus it makes it easy to find them all with a search. See https://stackoverflow.com/questions/9586478/ide-comment-keywords/9586701